pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "celero-bfm-fi-proto"
```

Se essa lib for só da Company acho q deveria mudar o nome
```suggestion
name = "celero-bfm-proto-company"
```